app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -199,16 +208,29 @@ public PopupWindow onButtonShowPopupWindowClick(View view, Boolean focusable, in
 
         return popupWindow;
     }
-
-
-
-//    @Override
-//    public boolean onNavigationItemSelected(@NonNull MenuItem item) {
-//        switch (item.getItemId()){
-//            case R.id.profile_button:
-////                getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new PlayerProfileFragment()).commit();
-//                break;
+    //TODO: fix it somehow: task is never completed and thus cannot get player from database
+    //To come back too later
+//    public void setPlayerObject(){
+//        playerId = getIntent().getIntExtra("playerId",0);
```

Comment:
Same here

Reply:
I would leave it here since it is something we will be using in the future

Reply:
Ok, that seems alright.